package.json

```diff
@@ -27,16 +27,20 @@
     "downshift": "2.2.3",
     "fs-extra": "7.0.0",
     "ramda": "0.25.0",
-    "react": "16.5.2",
-    "styled-components": "3.4.9",
     "tslint": "5.11.0",
     "typescript": "3.1.1"
```

Comment:
I think those two could be `devDependencies` too

Reply:
You're right